mkdocs_awesome_pages_plugin/navigation.py

```diff
@@ -1,5 +1,6 @@
 import warnings
 from pathlib import Path
+from natsort import os_sort_keygen
```

Comment:
Is it a good idea to use the `os_` variant of the function? I'm a bit hesitant because the behavior can change based on which OS the docs are built.

Reply:
Good catch. Gave some more detail in another comment further down the thread.